flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -471,6 +471,32 @@ class Table(
   }
 
   /**
+    * Intersect two [[Table]]s. Intersect All returns records that exist in both tables, but not
```

Comment:
I would rephrase this to "Intersect All returns records that exist in both tables. If a record is present in both tables more than once, it is returned as many times as it is present in both tables, i.e., the resulting table might have duplicate records."